index.html

```diff
@@ -383,7 +382,7 @@ <h3>Skills</h3>
 
     </section>
 
-    <!----------------- form area -->\
+    <!----------------- form area -->
```

Comment:
Some of the elements in your card are overlapping🥲 this can lead to a poor user experience🥲 Kindly give attention to it and make the necessary changes👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="300" alt="your card" src="https://user-images.githubusercontent.com/64643995/229760305-754dff72-b22b-45fd-aa98-25b2bef20207.png">|<img width="300" alt="design cards" src="https://user-images.githubusercontent.com/64643995/229760397-95fd9a8d-9ee7-4fd5-84bc-3a2ac85bb317.png">|

Reply:
hi [Ibizugbe](https://github.com/Ibizugbe)

please kindly check the the changes made again and give a comment